cypress/e2e/shared/influxCLI.test.ts

```diff
@@ -5,6 +5,7 @@ describe('Influx CLI onboarding', () => {
   })
 
   it('should load the main home-page that has the cli tile', function () {
+    cy.skipOn(isIOxOrg)
```

Comment:
```
const isIOxOrg = Boolean(Cypress.env('useIox'))
```
You may want to add this line to each test file since it is local to each test file

Reply:
updated